python/lsst/ip/isr/ptcDataset.py

```diff
@@ -171,16 +175,19 @@ class PhotonTransferCurveDataset(IsrCalib):
     attributes.
     Version 1.3 adds the `noiseMatrix` and `noiseMatrixNoB` attributes.
     Version 1.4 adds the `auxValues` attribute.
+    Version 1.5 adds the `covMatrixSideFullCovFit` attribute.
     """
 
     _OBSTYPE = 'PTC'
     _SCHEMA = 'Gen3 Photon Transfer Curve'
-    _VERSION = 1.4
+    _VERSION = 1.5
 
-    def __init__(self, ampNames=[], ptcFitType=None, covMatrixSide=1, **kwargs):
+    def __init__(self, ampNames=[], ptcFitType=None, covMatrixSide=1,
+                 covMatrixSideFullCovFit=1, **kwargs):
         self.ptcFitType = ptcFitType
         self.ampNames = ampNames
         self.covMatrixSide = covMatrixSide
+        self.covMatrixSideFullCovFit = covMatrixSideFullCovFit
```

Comment:
For maximum compatibility, could this default to `None` and use the value of `covMatrixSide` if it's none?  (It may not matter in usage).